src/instructlab/model/backends/backends.py

```diff
@@ -202,7 +203,8 @@ def ensure_server(
     host="localhost",
     port=8000,
     queue=None,
-) -> Tuple[multiprocessing.Process, str]:
+    server_process_func=None,
+) -> Tuple[multiprocessing.Process, subprocess.Popen, str]:
```

Comment:
@n1hility @danmcp I don't feel great about returning this tuple (one process for each backend) but in line with my comment one line aboe it felt like the least invasive thing I could do.

Reply:
this is good practice IMO could always improve the abstraction later